### PR TITLE
Bring application to foreground always when launched

### PR DESCRIPTION
**Pull request by rainemak**

### CODE REVIEW

**Comment by pvuorela** [comment]
```
void QWindow::show()
{
    Qt::WindowState defaultState = QGuiApplicationPrivate::platformIntegration()->defaultWindowState(d_func()->windowFlags);
    if (defaultState == Qt::WindowFullScreen)
        showFullScreen();
```

and qtwayland seem to have some ShowIsFullScreen stuff. If this has an effect maybe should also check Lipstick side?

File: rpm/libsailfishapp.spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Name: libsailfishapp
-Version: 0
+Version: 1
```
Comment (martyone) [inline rpm/libsailfishapp.spec]: I would prefer keeping zero here as better indication that the value is not accurate :)

**Comment by rainemak** [comment]
@pvuorela see https://git.merproject.org/mer-core/lipstick/merge_requests/74

**Comment by rainemak** [comment]
Lipstick cannot be fixed like suggested as it can cause issues with covers. Thus, would I'm favouring my original proposal as that does affect only QQuickView (app main window).

**Comment by pvuorela** [comment]
LGTM

**Pull request merged**